fuzzing/fuzzer.go

```diff
@@ -405,7 +406,22 @@ func chainSetupFromCompilations(fuzzer *Fuzzer, testChain *chain.TestChain) erro
 
 				// Ensure our transaction succeeded
 				if block.MessageResults[0].Receipt.Status != types.ReceiptStatusSuccessful {
-					return fmt.Errorf("contract deployment tx returned a failed status: %v", block.MessageResults[0].ExecutionResult.Err)
+					// Create a call sequence element to represent the failed contract deployment tx
+					cse := calls.NewCallSequenceElement(nil, msg, 0, 0)
+					cse.ChainReference = &calls.CallSequenceElementChainReference{
+						Block:            block,
+						TransactionIndex: len(block.Messages) - 1,
+					}
+					// Replay the execution trace for the failed contract deployment tx
+					err = cse.AttachExecutionTrace(testChain, fuzzer.contractDefinitions)
+					if err != nil {
+						fuzzer.logger.Warn("Failed to attach execution trace to failed contract deployment tx: ", err)
+					}
+					if cse.ExecutionTrace == nil {
+						return fmt.Errorf("contract deployment tx returned a failed status: %v", block.MessageResults[0].ExecutionResult.Err)
+					}
+
+					return fmt.Errorf("contract deployment tx returned a failed status: %v\n %s", block.MessageResults[0].ExecutionResult.Err, cse.ExecutionTrace.Log().String())
```

Comment:
This isn't as pretty as the normal traces, but it at least gives some context on where the revert is.